Monika After Story/game/script-topics.rpy

```diff
@@ -16300,3 +16300,30 @@ label monika_gmos:
     m 3esd "Like I said before, GMOs are a complex topic. {w=0.2}If you want to learn more, make sure that your sources are reliable and that you're able to see the discussion from both sides."
     m 1eua "I think that's enough for now, thanks for listening~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_favchocolate",
+            category=['misc'],
+            prompt="What's your favorite kind of chocolate?",
+            pool=True,
+        )
+    )
+
+label monika_favchocolate:
+    m 2hkb "Ooh, that's a tough question!"
+    m 2euu "I think if I had to pick, it would be dark chocolate."
+    m 3eub "It contains very little or no milk, so it has a less creamy texture, but a nice bittersweet taste."
+    m 3hub "Not to mention, it's rich in antioxidants and can even give some cardiovascular benefits! {w=0.3}{nw}"
+    extend 3hksdlb "...In moderation, of course."
+    m 1hubla "The taste kind of reminds me of a mocha coffee. Maybe the similarity in flavors is why I like it most."
+    m 1eud "Speaking of coffee, though...if I were drinking some, I think I might prefer milk or white chocolate for balance."
```

Comment:
We should probably probably check if she's currently drinking coffee/hotchoc. We can get the current drink via the existing consumables framework, so we should just need to check if it's coffee/hotchoc.

Reply:
I'm not sure how to code this, but I'd be happy to write variant dialogue for each case if we wanted to do this 👍

Reply:
You can get current drink via `MASConsumable._getCurrentDrink()`, and food via `MASConsumable._getCurrentFood()`. They'd return `None` if Monika doesn't drink/eat anything. For example, to compare it with coffee, you'd need to do this `MASConsumable._getCurrentDrink() is mas_getConsumable("coffee")`.